Yoast/wordpress-seo
# Indexation routine for post type archives

## Context


* We want to be able to index the post type archives when indexing the entire site using the indexation routine.

## Summary


This PR can be summarized in the following changelog entry:

* Adds indexable indexation of post type archives to the full site indexation routine.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Activate the Yoast Test Helper plugin. This is to make sure that you have some custom post types, namely 'book' and 'movie'.
* Run the `TRUNCATE TABLE wp_yoast_indexable` SQL query in your database to delete all pre-existing indexables from the indexable table.
* Go to _SEO_ > _Tools_ and click on the button to index your content.
* For every post type that has an archive page, an indexable should have been created in the indexable table with...
   * .. the `object_type` `post-type-archive`,
   * .. the `object_sub_type` having the value of the post type (e.g. `book`, `movie`).

## UI changes

## Documentation

## Quality assurance


Fixes #14850
